be/src/formats/parquet/group_reader.cpp

```diff
@@ -94,13 +101,6 @@ Status GroupReader::get_next(ChunkPtr* chunk, size_t* row_count) {
         }
     }
 
-    // dict filter chunk
-    {
-        SCOPED_RAW_TIMER(&_param.stats->expr_filter_ns);
-        SCOPED_RAW_TIMER(&_param.stats->group_dict_filter_ns);
-        has_filter = _dict_filter_ctx.filter_chunk(&active_chunk, &chunk_filter);
```

Comment:
```
has_filter |= _dict_filter_ctx.filter_chunk(&active_chunk, &chunk_filter);
```

Reply:
seems filter_chunk should return Status instead of bool, because `pred->evaluate` return Status